pkg/operator/staticpod/controller/installerstate/installer_state_controller.go

```diff
@@ -175,11 +175,17 @@ func (c *InstallerStateController) handlePendingInstallerPods(recorder events.Re
 				continue
 			}
 			if state := containerStatus.State.Waiting; len(state.Reason) > 0 {
+				message := fmt.Sprintf("Pod %q on node %q container %q is waiting for %s because", pod.Name, pod.Spec.NodeName, containerStatus.Name, pendingTime)
+				if len(state.Message) > 0 {
+					message = fmt.Sprintf("%s %q", message, state.Message)
+				} else {
+					message = fmt.Sprintf("%s %s", message, state.Reason)
+				}
```

Comment:
odd factoring.
```
reason := state.Reason
if len(state.Message) > 0 {
  reason = state.Message // prefer user-readible message
}
message := fmt.Sprintf("Pod %q on node %q container %q is waiting for %s because %q", pod.Name, pod.Spec.NodeName, containerStatus.Name, pendingTime, reason)
```
```